frontend/src/icons/Gear.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+type Props = {
+  width?: string;
+  fill?: string;
+};
+const Gear = ({width = '100%', fill = '#000'}: Props) => (
+  <svg
+    focusable="false"
+    role="img"
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 512 512"
+    width={width}
+  >
+    <path
+      fill={fill}
```

Comment:
is this an inline gear svg...

this is amaze

Reply:
I couldn't figure out how to import non-JS so everything is just JS 🙈